xml/docu_styleguide.structure.xml

```diff
@@ -889,7 +889,7 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
    HTML output.
   </para>
   <example xml:id="ex-figure">
-   <title>Example of a Figure</title>
+   <title>Example of a figure</title>
 <screen>&lt;figure xml:id="fig-picture"&gt;
  &lt;title&gt;An Interesting Picture&lt;/title&gt;
```

Comment:
Could you check for `example` titles that need fixing as well please?

Reply:
Bummer, I thought I'd caught them all. I'll definitely go through again.

Reply:
Sorry for the hassle. I guess it's not examples exactly, but DocBook source examples that are part of screens.